bootstrap.sh

```diff
@@ -143,7 +222,27 @@ cat <<EOF
 	$ git fetch origin
 
 	This may take some time depending on your internet connection speed.
+EOF
+		;;
+	clone-no-blobs)
+		cat <<EOF
+
+	Note that for performance reasons the server repository has been cloned with
+	--filter=blob:none. You have a complete history in the server repository.
+	If you checkout older commits, git will eventually download missing blobs on
+	the fly if they are not present locally.
 
+	You should be prepared to have a live internet connection when browsing the
```

Comment:
Is there a command to catch up the full history manually afterwards? Then we could also list that here.

Reply:
I have found [this link](https://docs.gitlab.com/ee/topics/git/partial_clone.html#remove-partial-clone-filtering). Should I add it in some way (a script?) or just document what would be needed to be done?

Reply:
I will have a look into this.

Reply:
I added a small script to unshallow/unpartial any existing repository. This script can be used rather generically.